started/index.md

```diff
@@ -139,10 +139,57 @@ The steps for installing with MSYS2 (recommended) are as follows:
 
 <div id="install__android" class="hidden">
 <div style="text-align: left" markdown="1">
-
-* To develop apps for Android you will first need to install the tools for your current computer (Windows, macOS or Linux)
+* To develop apps for Android you can use PC (Linux, Windows, Mac) or you can use Android app Termux (without PC)
+* To develop apps for Android on your computer (Windows, macOS or Linux) you need install android tools 
 * Once complete you will need to install the Android SDK and Android NDK - the recommended approach is to install [Android Studio](https://developer.android.com/studio/index.html) and then go to **Tools > SDK Manager** and from **SDK Tools** install the **NDK (Side by side)** package.
-
+* To develop apps for Android directly on Android app termux, you can follow this instructions
+1. it reqire android 9 above
+2. Install [fdroid](https://f-droid.org/) and from them [termux](https://f-droid.org/packages/com.termux/) and follow the instructions (or [video](https://youtu.be/uGtVjf4_Ivo))
+3. Install git:
+        $ pkg install git
+4. Download and run script:
+        $ cd ~ && git clone https://github.com/MatejMagat305/fyne-termux.git 
+        $ cd ~/fyne-termux && chmod 1777 fyne_install.sh
```

Comment:
We should not instruct people to execute shell scripts from third party repositories to get started - this could represent a big security risk!